cypress/e2e/person.js

```diff
@@ -27,7 +27,7 @@ describe('Merge person', () => {
     beforeEach(() => {
         cy.clickNavMenu('persons')
         cy.get('[data-attr=persons-search]').type('deb').should('have.value', 'deb')
-        cy.get('.ant-input-search-button').click()
+        cy.wait(1000)
```

Comment:
we should look for something in the DOM. Which is normally quicker than always waiting for 1 sec